src/main/java/edu/harvard/iq/dataverse/SendFeedbackDialog.java

```diff
@@ -101,8 +100,7 @@ public void initUserInput(ActionEvent ae) {
         op1 = Long.valueOf(random.nextInt(10));
         op2 = Long.valueOf(random.nextInt(10));
         userSum = null;
-        String supportEmail = JvmSettings.SUPPORT_EMAIL.lookupOptional().orElse(settingsService.getValueForKey(SettingsServiceBean.Key.SystemEmail));
-        systemAddress = MailUtil.parseSystemAddress(supportEmail);
+        systemAddress = mailService.getSupportAddress().orElse(null);
```

Comment:
Would it make sense to use the system email if no support address is provided instead of `null`?

Reply:
Maybe. For now, I only implemented a 100% backward compatible thing.